src/TestUtils.ts

```diff
@@ -9,12 +9,17 @@ import {
 
 export class TestLink extends ApolloLink {
     public operations: Operation[];
-    constructor() {
+    public onRequest: Function;
```

Comment:
If you make the changes I suggested in the test, then I don't think you need any of the changes in this file either.